### PR TITLE
[17.0][MIG] base_external_dbsource_sqlite

### PR DESCRIPTION
**Pull request by dalonsod**
Standard migration, depends on #292

### CODE REVIEW

File: base_external_dbsource_sqlite/demo/base_external_dbsource.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<odoo>
```
Comment (carolinafernandez-tecnativa) [inline base_external_dbsource_sqlite/demo/base_external_dbsource.xml]: Please add noupdate="1"

**Review by carolinafernandez-tecnativa — COMMENTED** [review]
Please squash this administrative commit:
image

Could you please check test are failing and runbot too.

Thanks!

**Comment by carolinafernandez-tecnativa** [comment]
Hi could you please check my previous comments so can we move forward to this migration? Thanks!!

**Comment by dalonsod** [comment]
@Anxo82 could you make the two changes requested by @carolinafernandez-tecnativa and add a temporary `test-requirements.txt` that points to #292?

**Review by carolinafernandez-tecnativa — APPROVED** [review]
LGTM Thanks :)

**Comment by carolinafernandez-tecnativa** [comment]
ping @pedrobaeza @sergio-teruel

**Comment by carolinafernandez-tecnativa** [comment]
Could you please rebase? base_external_dbsource is already merged.

**Comment by dalonsod** [comment]
> Could you please rebase? base_external_dbsource is already merged.

Done, and also a rebase with `origin/17.0`

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**